client/src/components/ItemDetails/itemdetails.style.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const ImgDiv = styled.div`
+  position: absolute;
+  display: flex;
+  justify-content: flex-end;
+  z-index: -1;
```

Comment:
What is the z-index doing here?

Reply:
to get the delete icon in front of the form component